kahuna/public/js/leases/leases.css

```diff
@@ -93,14 +111,41 @@ gr-leases .gu-date-range__display {
 
 .lease__access {
   padding: 3px;
-  margin-bottom: 7px;
+  margin-bottom: 2px;
 }
 
 .lease__current {
   padding: 2px;
+}
 
+.lease__current.current {
+  background-color: #00ADEE;
 }
 
 .current {
-  background-color: #00ADEE;
-}
\ No newline at end of file
+  color: white;
+}
+
+gr-leases.leases-flex {
+  display: flex;
+  flex-direction: row;
+}
+
+gr-leases.leases-flex ul {
+  display:flex;
+  flex-direction: row;
+  line-height: 1em;
+  font-size: .75em;
```

Comment:
Mixing between `rem` and `em` is bad. tru to use just one, probably `rem` as it's far easier to predict what the result will be.